File: Example/CoreData/SyncKitCoreDataExample/Model/QSCompany+CoreDataProperties.m
//
//  QSCompany+CoreDataProperties.m
//  SyncKitCoreDataExample
//
//  Created by Manuel Entrena on 08/05/2018.
//  Copyright © 2018 Manuel Entrena. All rights reserved.
//
//

#import "QSCompany+CoreDataProperties.h"

@implementation QSCompany (CoreDataProperties)

+ (NSFetchRequest<QSCompany *> *)fetchRequest {
	return [NSFetchRequest fetchRequestWithEntityName:@"QSCompany"];
}

@dynamic identifier;
@dynamic name;
@dynamic sortIndex;
@dynamic employees;

@end

File: Example/CoreData/SyncKitCoreDataExample/SyncKitCoreDataExample/Model/CoreData/QSCompany+CoreDataProperties.swift
//
//  QSCompany+CoreDataProperties.swift
//  
//
//  Created by Manuel Entrena on 11/06/2019.
//
//

import Foundation
import CoreData


extension QSCompany {

    @nonobjc public class func fetchRequest() -> NSFetchRequest<QSCompany> {
        return NSFetchRequest<QSCompany>(entityName: "QSCompany")
    }

    @NSManaged public var identifier: String?
    @NSManaged public var name: String?
    @NSManaged public var sortIndex: NSNumber?
    @NSManaged public var employees: NSSet?

}

// MARK: Generated accessors for employees
extension QSCompany {

    @objc(addEmployeesObject:)
    @NSManaged public func addToEmployees(_ value: QSEmployee)

    @objc(removeEmployeesObject:)
    @NSManaged public func removeFromEmployees(_ value: QSEmployee)

    @objc(addEmployees:)
    @NSManaged public func addToEmployees(_ values: NSSet)

    @objc(removeEmployees:)
    @NSManaged public func removeFromEmployees(_ values: NSSet)

}
